infrastructure/main.tf

```diff
@@ -81,6 +81,16 @@ data "azurerm_key_vault_secret" "s2s_microservice" {
   vault_uri = "${data.azurerm_key_vault.ia_key_vault.vault_uri}"
 }
 
+data "azurerm_key_vault_secret" "ia_customer_services_telephone" {
```

Comment:
I am not sure if we need to add new variables to main.tf file. All envs are in AKS now. It is fine to leave then here